integration_test_project/models/tests_and_exposures.yml

```diff
@@ -10,10 +10,10 @@ models:
           - unique
 
 exposures:
-  - name: "ceo's imaginary dashboard"
+  - name: "ceos_imaginary_dashboard"
```

Comment:
What's the thinking here? :)

Reply:
dbt 1.3 gave a warning that spaces in the name will not be permitted in a future version. Also with sql server the spaces were failing the ci tests, so killed two birds with one stone.